documentation/dsl.md

```diff
@@ -784,6 +783,28 @@ Addresses.CALCULATE(state, city).WHERE(
 People.WHERE(PERCENTILE(by=COUNT(packages).ASC()) == 100)
 ```
 
+**Good Example #7**: Sort every person by their first name. This is valid because the collation term is by default ascending based on the `collation_default_asc` config.
+
+```py
+%%pydough
+People.ORDER_BY(first_name)
+```
+
+**Good Example #8**: Sort every person by their first name in ascending order, last name in descending order, and the number of packages they have ordered in descending order.
+Let's keep the default behavior of `collation_default_asc` and set `propogate_collation` to `True`.
+
+```py
+%%pydough
+People.ORDER_BY(first_name, last_name.DESC(), COUNT(packages))
+```
+
+This is valid because the collation term is by default ascending based on the `collation_default_asc` config. Setting the `propogate_collation` config to `True` will cause the collation to be propogated to the `COUNT(packages)` term. Hence its equivalent to:
```

Comment:
To avoid confusion be explicit that it will get it from last_name.
e.g. 
`will cause the collation to be propogated to the `COUNT(packages)` term from last used one (`last_name.DESC()`) .